CMakeLists.txt

```diff
@@ -736,6 +742,11 @@ install(
         "${CMAKE_CURRENT_SOURCE_DIR}/README.md"
   DESTINATION "${CMAKE_INSTALL_DOCDIR}")
 
+
+# -- packaging -----------------------------------------------------------------
+
+include(Package)
```

Comment:
The current custom tar.gz thing is broken because the nix build doesn't do a regular install anymore. The way to go is to use CPack to create that as well. Once we have component based installation the output should be equivalent.

Reply:
I changed things so we now get a `.tar.zst` from CPack.

Reply:
`git grep \.tar\.gz` still yields quite a few links that will be broken after the next release with this merged. Are you sure we don't just wanna stick with the more widely available `.tar.gz`?

Reply:
Reverted to tar.gz for link stability. 
Although `zst` is commonly accepted in most places nowadays.